tests/test_responsetypes.py

```diff
@@ -79,10 +68,18 @@ def test_from_args(self):
             # headers takes precedence over url
```

Comment:
You can remove the TODO above. Now that we are relying on a third-party library for MIME sniffing, that is no longer a concern of ours, it is xtractmime tests that should cover that.

The comment in this line can be removed as well.